File: examples/python/hello.py
#!/usr/bin/env python3
"""
Sample Python script for Git practice.

This script demonstrates a simple greeting function.
You can modify this file to practice Git commands.
"""

def greet(name):
    """Return a greeting message."""
    return f"Hello, {name}! Welcome to Git practice."

def main():
    """Main function."""
    user_name = "Git Learner"
    message = greet(user_name)
    print(message)

if __name__ == "__main__":
    main()

File: examples/javascript/hello.js
// Sample JavaScript file for Git practice

/**
 * Greet a user with a personalized message
 * @param {string} name - The name of the user
 * @returns {string} A greeting message
 */
function greet(name) {
    return `Hello, ${name}! Welcome to Git practice.`;
}

/**
 * Main function to demonstrate the greeting
 */
function main() {
    const userName = "Git Learner";
    const message = greet(userName);
    console.log(message);
}

// Run the main function
main();
